docs/content/posts/_index.md

```diff
@@ -1,4 +0,0 @@
----
```

Comment:
Blog infra has been removed for now. We can always add this in the future as needed.